types/aws-lambda/trigger/appsync-resolver.d.ts

```diff
@@ -11,12 +11,24 @@ export type AppSyncBatchResolverHandler<TArguments, TResult, TSource = Record<st
     TResult[]
 >;
 
-// See https://docs.aws.amazon.com/appsync/latest/devguide/security-authz.html#aws-lambda-authorization
+/**
+ * @deprecated Use AppSyncAuthorizerHandler (fixes typo)
+ */
 export type AppSyncAuthorizerHander<TResolverContext = undefined> = Handler<
     AppSyncAuthorizerEvent,
     AppSyncAuthorizerResult<TResolverContext>
 >;
```

Comment:
You could point it at the other, which makes it a bit clearer and shorter.

Reply:
This what you mean?

Reply:
More a case of DRY I think:
```ts
export type AppSyncAuthorizerHandler<TResolverContext = undefined> = Handler<
    AppSyncAuthorizerEvent,
    AppSyncAuthorizerResult<TResolverContext>
>;

/**
 * @deprecated Use {@link AppSyncAuthorizerHandler}
*/
export type AppSyncAuthorizerHander<TResolverContext = undefined> = AppSyncAuthorizerHandler<TResolverContext>;
```

Reply:
Ahh ... sure! Pushed an new commit